src/AutoRest.CSharp.V3/Generation/Writers/ManagementClientWriter.cs

```diff
@@ -80,6 +90,17 @@ public static void WriteAggregateClient(CodeWriter writer, BuildContext context)
                     }
 
                     writer.WriteXmlDocumentationSummary($"Initializes a new instance of {title}ManagementClient");
+                    foreach (Parameter parameter in allParameters.Values)
+                    {
+                        writer.WriteXmlDocumentationParameter(parameter.Name, parameter.Description);
+                    }
+                    writer.WriteXmlDocumentationParameter("tokenCredential", "The OAuth token for making client requests.");
+                    writer.WriteXmlDocumentationParameter("options", "The options for configuring the client.");
+                    if (allParameters.Values.HasAnyNullCheck())
```

Comment:
I wonder if we should just hardcode this to always be printed

Reply:
Since it is based on the number of parameters in a constructor, there won't ever be more than a handful of parameters. So, the check for this is very fast. I think when there would be multiple checks like this, you'd need a different strategy. But, having one for now shouldn't be an issue.